src/components/modal/report/modal.jsx

```diff
@@ -124,120 +79,88 @@ class ReportModal extends React.Component {
-                                            minLength: this.props.intl.formatMessage({id: 'report.tooShortError'})
-                                        }}
-                                        validations={{
-                                            maxLength: 500,
-                                            minLength: 20
+                    <ModalInnerContent className="report-modal-content">
+                        {isError && (
+                            <div className="error-text">
+                                <FormattedMessage id="report.error" />
+                            </div>
+                        )}
+                        <Progression step={isConfirmed ? STEPS.confirmation : this.state.step}>
+                            <FormStep
+                                nextLabel={{id: 'general.next'}}
+                                onNext={this.handleSetCategory}
+                            >
+                                <div className="instructions">
+                                    <FormattedMessage
+                                        id={`report.${type}Instructions`}
+                                        key={`report.${type}Instructions`}
+                                        values={{
+                                            CommunityGuidelinesLink: (
+                                                <a href="/community_guidelines">
+                                                    <FormattedMessage id="report.CommunityGuidelinesLinkText" />
+                                                </a>
+                                            )
                                         }}
-                                        value={this.state.notes}
                                     />
                                 </div>
-                            )}
-                            {isError && (
-                                <div className="error-text">
-                                    <FormattedMessage id="report.error" />
+                                <Select
+                                    required
+                                    elementWrapperClassName="report-modal-field"
+                                    label={null}
+                                    name="category"
+                                    options={reportOptions.map(option => ({
+                                        value: option.value,
+                                        label: intl.formatMessage(option.label),
+                                        key: option.value
+                                    }))}
+                                    validationErrors={{
+                                        isDefaultRequiredValue: categoryRequiredMessage
+                                    }}
+                                />
+                            </FormStep>
+                            <FormStep
```

Comment:
Could you put a comment before this line, making clear that it constitutes the content of the `textInput` step?

Reply:
Done